### PR TITLE
Fix issue 719

### PR DESCRIPTION
**Pull request by n063h**
Fixes  ISSUE#719

Changes:

1. allow users to customize pulsar env
2. allow users to customize pulsar confs
3. fix sslPort format error

Before

install cluster `kubefate cluster install -f  examples/party-10000/cluster-spark-local-pulsar.yaml`
run `❯ k edit pod -n fate-10000 pulsar-0`, you can see
```
spec:
  containers:
  - command:
    - /bin/bash
    - -c
    - bin/pulsar standalone -nss
    image: federatedai/pulsar:2.7.0
```
And run `kexn  fate-10000 -it  pulsar-0 bash`, you can see in /pulsar/conf/standalone.conf
```
# Default per-topic backlog quota limit
backlogQuotaDefaultLimitGB=10

# How often to check for inactive topics
brokerDeleteInactiveTopicsFrequencySeconds=60
```

After
User can customize pulsar env and confs in `examples/party-10000/cluster-spark-local-pulsar.yaml`, like
```
  env:
  - name: PULSAR_MEM
    value: "-Xms4g -Xmx4g -XX:MaxDirectMemorySize=8g"
  confs:
    brokerDeleteInactiveTopicsFrequencySeconds: 10086
    backlogQuotaDefaultLimitGB: 10010
```
then run `❯ k edit pod -n fate-10000 pulsar-0`, you can see
```
    env:
    - name: PULSAR_MEM
      value: -Xms4g -Xmx4g -XX:MaxDirectMemorySize=8g
    image: federatedai/pulsar:2.7.0
    imagePullPolicy: IfNotPresent
```
run `❯ kexn  fate-10000 -it  pulsar-0 bash`, you can see
```
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
root@pulsar-0:/pulsar# echo $PULSAR_MEM
-Xms4g -Xmx4g -XX:MaxDirectMemorySize=8g
```
And you can see in /pulsar/conf/standalone.conf
```
# Default per-topic backlog quota limit
backlogQuotaDefaultLimitGB=10010

# How often to check for inactive topics
brokerDeleteInactiveTopicsFrequencySeconds=10086
```
It turns out the env and confs have been set inside container.

### CODE REVIEW

**Comment by JingChen23** [comment]
There should be another 2 configurations?

https://github.com/FederatedAI/KubeFATE/pull/715/files

We also need to varibalize the 2 configurations in pulsar's configmap, make them in the cluster.yaml

**Comment by n063h** [comment]
> There should be another 2 configurations?
> 
> https://github.com/FederatedAI/KubeFATE/pull/715/files
> 
> We also need to varibalize the 2 configurations in pulsar's configmap, make them in the cluster.yaml

It's done.

File: k8s-deploy/examples/party-10000/cluster-spark-local-pulsar.yaml [hunk 1]
```diff
@@ -65,6 +65,8 @@ pulsar:
   type: NodePort
   httpNodePort: 30104
   httpsNodePort: 30109
+  env:
+  confs:
```
Comment (JingChen23) [inline k8s-deploy/examples/party-10000/cluster-spark-local-pulsar.yaml:69]: In examples, we don't write useless stuffs. I suggest remove them and update the document:

https://github.com/FederatedAI/KubeFATE/blob/develop-1.9.0/docs/configurations/FATE_cluster_configuration.md

I find that this document has no configration information for pulsar. So please help to add all the pulsar related information into this file, thank you very much!!

File: k8s-deploy/examples/party-10000/cluster-spark-pulsar.yaml [hunk 2]
```diff
@@ -67,6 +67,8 @@ pulsar:
   type: NodePort
   httpNodePort: 30104
   httpsNodePort: 30109
+  env:
+  confs:
```
Comment (JingChen23) [inline k8s-deploy/examples/party-10000/cluster-spark-pulsar.yaml:71]: ditto.

File: k8s-deploy/examples/party-9999/cluster-spark-local-pulsar.yaml [hunk 3]
```diff
@@ -65,6 +65,8 @@ pulsar:
   type: NodePort
   httpNodePort: 30094
   httpsNodePort: 30099
+  env:
+  confs:
```
Comment (JingChen23) [inline k8s-deploy/examples/party-9999/cluster-spark-local-pulsar.yaml:69]: ditto

File: k8s-deploy/cluster-spark-pulsar.yaml [hunk 4]
```diff
@@ -233,6 +233,7 @@ device: CPU
   # existingClaim:
   # accessMode: ReadWriteOnce
   # size: 1Gi
+  # env:
```
Comment (JingChen23) [inline k8s-deploy/cluster-spark-pulsar.yaml:236]: Add `conf:` below this line?

**Review by JingChen23 — CHANGES_REQUESTED** [review]
Overall looks good, just need to help change another doc, thanks!

**Review by JingChen23 — APPROVED** [review]

**Pull request merged**